src/test/java/uk/gov/digital/ho/pttg/api/IncomeSummaryServiceTest.java

```diff
@@ -346,19 +342,103 @@ public void shouldLogInfoWhenRetryingApiCallOnUnexpectedError() {
             // Ignore expected exception
         }
 
-        verifyHmrcCallMessage("HMRC call attempt 1 of 5");
-        verifyHmrcCallMessage("HMRC call attempt 2 of 5");
-        verifyHmrcCallMessage("HMRC call attempt 3 of 5");
-        verifyHmrcCallMessage("HMRC call attempt 4 of 5");
-        verifyHmrcCallMessage("HMRC call attempt 5 of 5");
+        verifyHmrcCallMessage("HMRC call attempt 1");
+        verifyHmrcCallMessage("HMRC call attempt 2");
+        verifyHmrcCallMessage("HMRC call attempt 3");
+        verifyHmrcCallMessage("HMRC call attempt 4");
+        verifyHmrcCallMessage("HMRC call attempt 5");
+    }
+
+    @Test
+    public void shouldNotProduceIncomeSummaryWhenSufficientRetriesButInsufficientTime() {
+
+        RequestHeaderData mockRequestHeaderData = mock(RequestHeaderData.class);
```

Comment:
You don't need a local version - you can use the field

Reply:
Removed the variable to reuse the field version.